src/SimpleGraphs/generators/randgraphs.jl

```diff
@@ -989,6 +989,144 @@ function uniform_tree(n::Integer; rng::Union{Nothing,AbstractRNG}=nothing)
     return prufer_decode(random_code)
 end
 
+"""
```

Comment:
I wonder if `randgraphs.jl` is the correct place for these functions, maybe `staticgraphs.jl` is a better place, as generators are not random.

Reply:
We moved it to `staticgraphs.jl`. Thanks for the suggestion.